### PR TITLE
docs: hexo-util@1 & hexo-renderer-marked@2

### PR DESCRIPTION
**Pull request by curbengh**
## Check List

**Please read and check followings before submitting a PR.**

- [x] Others (Update, fix, translation, etc...)

https://github.com/hexojs/hexo-util/releases/tag/1.0.0
https://github.com/hexojs/hexo-renderer-marked/releases/tag/2.0.0

### CODE REVIEW

File: source/_posts/2019-08-21-hexo-util-1-0-0_hexo-renderer-marked-2-0-0_released.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+[#36]: https://github.com/hexojs/hexo-util/pull/36
+[#69]: https://github.com/hexojs/hexo-util/pull/69
+[#67]: https://github.com/hexojs/hexo-util/pull/67
+[#47]: https://github.com/hexojs/hexo-util/pull/47
+[#48]: https://github.com/hexojs/hexo-util/pull/48
+[#44]: https://github.com/hexojs/hexo-util/pull/44
+[#46]: https://github.com/hexojs/hexo-util/pull/46
+[#70]: https://github.com/hexojs/hexo-util/pull/70
+
+[#98]: https://github.com/hexojs/hexo-renderer-marked/pull/70
```
Comment (YoshinoriN) [inline source/_posts/2019-08-21-hexo-util-1-0-0_hexo-renderer-marked-2-0-0_released.md]:
```diff
-[#98]: https://github.com/hexojs/hexo-renderer-marked/pull/70
+[#98]: https://github.com/hexojs/hexo-renderer-marked/pull/98
```

Comment (curbengh) [reply source/_posts/2019-08-21-hexo-util-1-0-0_hexo-renderer-marked-2-0-0_released.md]: Ops. Fixed.

**Comment by curbengh** [comment]
Please squash and merge. :wink:

**Comment by SukkaW** [comment]
LGTM.

**Pull request merged**